cloudify/models_states.py

```diff
@@ -93,3 +93,13 @@ class AgentState(object):
     STATES = [CREATING, CREATED, CONFIGURING, CONFIGURED, STARTING, STARTED,
               STOPPING, STOPPED, DELETING, DELETED, RESTARTING, RESTARTED,
               UPGRADING, UPGRADED, FAILED, NONRESPONSIVE, RESTORED]
+
+
+class PluginInstallationState(object):
+    PENDING = 'pending'
```

Comment:
Would it makes sense to have this be pending-install, to mirror pending-uninstall?